mne/io/base.py

```diff
@@ -658,7 +658,7 @@ def filenames(self):
         return tuple(self._filenames)
 
     @annotations.setter
-    def annotations(self, annotations):
+    def annotations(self, annotations, emit_warning=True):
```

Comment:
It's a setter. This gets called by ``raw.annotations = annotations``. This is only exposed if for some reason the user wants to call it with ``mne,BaseRaw.annotations.fset(...)`` (which they shouldn't do).